src/mesh/domain.hpp

```diff
@@ -140,6 +142,23 @@ class IndexShape {
     }
   }
 
+  KOKKOS_INLINE_FUNCTION
+  static IndexShape GetFromSeparateInts(int nx3, int nx2, int nx1, int ng) {
+    IndexShape out;
+    int idx = 0;
+    for (auto nx : {nx1, nx2, nx3}) {
+      if (nx == 0) {
```

Comment:
Do we actually have this case? nx1, nx2 and and nx3 are typically >=1, aren't they?

Reply:
Added a brief comment.

Agreed that we should do a documentation sprint at some point in the near future.